src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  let myList = list;
+
+  myList = document.querySelector('ul');
+
+  const salaryElement = myList.querySelector('data-salary');
+
+  return salaryElement;
+}
+
+sortList();
```

Comment:
The `sortList` function is called without any arguments, which will lead to errors since the function expects a parameter. Pass the appropriate list to be sorted as an argument.